.travis.yml

```diff
@@ -23,3 +23,15 @@ script:
 after_success:
   - SBT_OPTS=-J-Xmx3g sbt "project http" coveralls
   - docker rm -f mysql
+
+# https://www.scala-sbt.org/1.x/docs/Travis-CI-with-sbt.html describes how to enable caches for sbt dependencies
+before_cache:
```

Comment:
Just curious what prompted this change

Reply:
this is from some dependencies keep failing to resolve...I copied this from what David An did for orchestration